aws/modules/infrastructure_modules/stacks_app/example/main.tf

```diff
@@ -0,0 +1,28 @@
+
+provider "aws" {
+}
+
+resource "random_pet" "this" {
```

Comment:
Is it worth renaming this resource to be more specific? Also do we care is it's a mix of alphanumeric characters? If so, we can limit this in the resource to just numbers etc

Reply:
Yes this is not the real manifest that will be consumed, it's was just for testing out, I have in mind to write more clear examples so for time being left that out, but i can update it to random strings then pet names :-)